server/mediaserver.go

```diff
@@ -1626,6 +1629,9 @@ func (s *LivepeerServer) GetNodeStatus() *common.NodeStatus {
 			res.OrchestratorPool = append(res.OrchestratorPool, info.URL.String())
 		}
 	}
+	if s.LivepeerNode.NodeType.String() == "orchestrator" {
```

Comment:
Don't we want to display it in `livepeer_cli`?

Reply:
Removed if statement